frontend/components/modal/Error.vue

```diff
@@ -1,14 +1,46 @@
 <script setup lang="ts">
-const { error: { errorModalVisible, error } } = useModal()
+import { watch } from 'vue'
+
+interface Props {
+  isFullPage?: boolean
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  isFullPage: false
+})
+
+const { error: { errorModalVisible, error: modalError } } = useModal()
+const error = useError()
+const router = useRouter()
+
+watch(errorModalVisible, async (newV, oldV) => {
+  if (!props.isFullPage) {
+    return
+  }
+  // We only care about the case where the modal was just closed (i.e. has gone from visible -> not visible).
```

Comment:
I try to avoid watch like the plague - since this is event driven I think you can capture this same behavior using the [@closed event](https://github.com/RMI-PACTA/app/blob/2525f7f39e0748935a0f17b1174aa717a469c0da/frontend/components/standard/Modal.vue#L25)